crates/examples/src/readobj/macho.rs

```diff
@@ -378,6 +382,9 @@ fn print_load_command<Mach: MachHeader>(
             LoadCommandVariant::Symtab(symtab) => {
                 print_symtab::<Mach>(p, endian, state.linkedit_data, symtab, state);
             }
+            LoadCommandVariant::LinkeditData(linkedit) => {
```

Comment:
Move this down into the next `match variant`.

Reply:
With the new flag, I'm not sure if this should still be moved down. But I think it makes sense for the function starts to be nested in the associated load command, so both need to be displayed via the same match.